lib/AST/NameLookup.cpp

```diff
@@ -28,6 +28,11 @@
 #include "llvm/ADT/DenseMap.h"
 #include "llvm/ADT/TinyPtrVector.h"
 
+#if defined(_MSC_VER) && !defined(__clang__)
```

Comment:
Oh, I see, it's the popcnt check below. I think it's fine to leave that out, since the static_assert will be checked in Clang anyway.